hydra-tui/src/Hydra/TUI/Forms.hs

```diff
@@ -60,3 +60,23 @@ utxoRadioField u =
         ]
     ]
     (Prelude.head $ Map.toList u)
+
+confirmRadioField ::
+  forall s e n.
+  ( s ~ Bool
+  , n ~ Text
+  ) =>
+  Form s e n
+confirmRadioField =
+  newForm
+    [ radioField
+        id
+        [ (snd opt, fst opt, fst opt)
+        | opt <- options
+        ]
+    ]
+    True
+ where
+  options = [("[y]es", True), ("[n]o", False)]
```

Comment:
These options indicate we should be pressing `y` or `n`. However, the event handler for `ConfirmingAbort` is matching `KEsc` and `KEnter`?

Reply:
yes, it was a bad idea to use `y` and `n`, thus the reason i change the key binding, although i forgot to update the labels... good catch!